components/klaudsolcms/modals/AddEditAnotherFieldModal.js

```diff
@@ -1,53 +1,77 @@
-import AppModal from '@/components/klaudsolcms/AppModal';
-import {Formik, Form, Field} from 'formik';
+import AppModal from "@/components/klaudsolcms/AppModal";
+import { Formik, Form, Field } from "formik";
 
-export const ADD_MODE = 'ADD_MODE';
-export const EDIT_MODE = 'EDIT_MODE';
+export const ADD_MODE = "ADD_MODE";
+export const EDIT_MODE = "EDIT_MODE";
 
-export default function AddEditAnotherFieldModal({formParams, show, onClose, onClick, mode}) {
+const input = [
```

Comment:
Ideally, this should be coming from the backend. But for now, you can refactor this and move it to a separate file named `constants.js` The file should look like this.

```suggestion
export const inputValues = [
  { value: "text", option: "Text"},
  { value: "textarea", option: "Text Area"},
  { value: "link", option: "Link"},
  { value: "image", option: "Image"},
  { value: "float", option: "Number"},
]
```

then import it in this file.